lib_forceload/data/gm4_forceload-1.0/functions/load.mcfunction

```diff
@@ -1,6 +1,6 @@
 scoreboard objectives add gm4_dimension dummy
 
-execute in minecraft:overworld unless block 29999998 1 7133 birch_wall_sign run function gm4_forceload-1.0:init_chunk
+execute store success score $initialized gm4_dimension if block 29999998 1 7133 birch_wall_sign
+execute if score $initialized gm4_dimension matches 0 run summon marker ~ 0 ~ {CustomName:'"minecraft:overworld"',Tags:["gm4_dimension","gm4_new_dimension"]}
+execute if score $initialized gm4_dimension matches 0 as @e[type=marker,tag=gm4_new_dimension,limit=1] run function gm4_forceload-1.0:init_ow_chunk
 function gm4_forceload-1.0:load_wait
-
-execute as @a at @s run function gm4_forceload-1.0:check_dimension
```

Comment:
What's the reason that the overworld needs a custom treatment rather than how it was done before?

Reply:
Since we're now checking the block instead of the marker, setting the overworld sign will cause the overworld marker to never generate.